src/components/common/error/ErrorBar.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { NavLink, useHistory } from 'react-router-dom'
+import { URLS } from '../../../config'
+import { ReactComponent as ErrorInfo } from '../../../assets/icons/misc/errorInfo.svg'
+import { AppDetails } from '../../v2/appDetails/appDetails.type'
+
+function ErrorBar({ appDetails }: { appDetails: AppDetails }) {
```

Comment:
Move to types & write `export default` here itself as you just have a single export from here.